### PR TITLE
added OMF Editor info tab content

### PR DESCRIPTION
**Pull request by jasnames**
The original info tab content can be found in the PBI. I edited to remove passive voice, add where you find the editor, and update the command access.

### CODE REVIEW

**Review by osisoft-mbishop — APPROVED** [review]
looks good!

File: content/developer-guide/omf-dev/omf-editor.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+uid: omfEditor
+---
+
+# OMF editor
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:5]:
```suggestion
# OMF editor

```
Needs a blank line after for Zoomin formatting.

File: content/developer-guide/omf-dev/omf-editor.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+The Open Message Format (OMF) defines a set of message headers and message bodies. You write messages in JSON format that generate compliant messages for data ingress. The OMF specification is generic in that it does not specify a particular back-end system. You can use OMF to create types, create streams, and populate streams with data.
+
+The OMF Editor, found at **Developer Tools** > **OMF Editor**, helps build and validate OMF messages to be ingested by the Sequential Data Store.
+There are three message formats you can use to accomplish these tasks: type messages, container messages, and data messages. In the editor choose the message type from the **OMF Type** selector: **Type**, **Container** or **Data**.
+
+You can edit your message and utilize the following options for each message format provided:
+
+- Click **Copy OMF** to copy the message content to your clipboard.
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:13]:
```suggestion
- Select **Copy OMF** to copy the message content to the clipboard. 

```

File: content/developer-guide/omf-dev/omf-editor.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# OMF editor
+The Open Message Format (OMF) defines a set of message headers and message bodies. You write messages in JSON format that generate compliant messages for data ingress. The OMF specification is generic in that it does not specify a particular back-end system. You can use OMF to create types, create streams, and populate streams with data.
+
+The OMF Editor, found at **Developer Tools** > **OMF Editor**, helps build and validate OMF messages to be ingested by the Sequential Data Store.
+There are three message formats you can use to accomplish these tasks: type messages, container messages, and data messages. In the editor choose the message type from the **OMF Type** selector: **Type**, **Container** or **Data**.
+
+You can edit your message and utilize the following options for each message format provided:
+
+- Click **Copy OMF** to copy the message content to your clipboard. 
+- Click **More Options** ![More Options](..\images\more-options.png "More Options") and **Format OMF** to reset template content alignment after making changes.
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:14]:
```suggestion
- Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Format OMF** to reset template content alignment after making changes. 

```

File: content/developer-guide/omf-dev/omf-editor.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+The OMF Editor, found at **Developer Tools** > **OMF Editor**, helps build and validate OMF messages to be ingested by the Sequential Data Store.
+There are three message formats you can use to accomplish these tasks: type messages, container messages, and data messages. In the editor choose the message type from the **OMF Type** selector: **Type**, **Container** or **Data**.
+
+You can edit your message and utilize the following options for each message format provided:
+
+- Click **Copy OMF** to copy the message content to your clipboard. 
+- Click **More Options** ![More Options](..\images\more-options.png "More Options") and **Format OMF** to reset template content alignment after making changes. 
+- Click **More Options** ![More Options](..\images\more-options.png "More Options") and **Download OMF** to download the message content to your local drive.
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:15]:
```suggestion
- Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Download OMF** to download the message content to your local drive. 

```

File: content/developer-guide/omf-dev/omf-editor.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+The OMF Editor, found at **Developer Tools** > **OMF Editor**, helps build and validate OMF messages to be ingested by the Sequential Data Store.
+There are three message formats you can use to accomplish these tasks: type messages, container messages, and data messages. In the editor choose the message type from the **OMF Type** selector: **Type**, **Container** or **Data**.
+
+You can edit your message and utilize the following options for each message format provided:
+
+- Click **Copy OMF** to copy the message content to your clipboard. 
+- Click **More Options** ![More Options](..\images\more-options.png "More Options") and **Format OMF** to reset template content alignment after making changes. 
+- Click **More Options** ![More Options](..\images\more-options.png "More Options") and **Download OMF** to download the message content to your local drive. 
+- Click **More Options** ![More Options](..\images\more-options.png "More Options") and **Reset Template** to delete any changes you have made to the message content.
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:16]:
```suggestion
- Select **More Options** ![More Options](..\images\more-options.png "More Options") > **Reset Template** to delete any changes you have made to the message content. 

```

File: content/developer-guide/omf-dev/omf-editor.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+uid: omfEditor
+---
+
+# OMF editor
+The Open Message Format (OMF) defines a set of message headers and message bodies. You write messages in JSON format that generate compliant messages for data ingress. The OMF specification is generic in that it does not specify a particular back-end system. You can use OMF to create types, create streams, and populate streams with data.
+
+The OMF Editor, found at **Developer Tools** > **OMF Editor**, helps build and validate OMF messages to be ingested by the Sequential Data Store.
```
Comment (afloresOSI) [inline content/developer-guide/omf-dev/omf-editor.md:8]: Paragraphs need to be separated by a blank line. 
```suggestion
To access the OMF Editor, select **Developer Tools** > **OMF Editor** from the left pane. Use the editor to build and validate OMF messages to send to the Sequential Data Store.

```

**Review by afloresOSI — CHANGES_REQUESTED** [review]
For formatting in Zoomin, add a blank line after:
- the title line
- each bullet line

**Review by afloresOSI — APPROVED** [review]

**Pull request merged**